admin/src/components/Dashboard/Widgets/__snapshots__/ControllerConsentDial.test.js.snap

```diff
@@ -7,15 +7,15 @@ exports[`(Component) ConsentDial should render 1`] = `
     Array [
       Object {
         "x": "Consented",
-        "y": undefined,
+        "y": 1,
       },
       Object {
         "x": "Not Consented",
-        "y": undefined,
+        "y": 10,
       },
     ]
   }
-  text="100% of subjects have consented"
+  text="1100% of subjects have consented"
```

Comment:
That is obviously wrong it should be 9% (1/11), but I fixed this and it's already on master. Should be updated after rebase.